google-cloud-core/resources/META-INF/google-cloud-core.xml

```diff
@@ -61,23 +61,9 @@
                         serviceImplementation="com.google.cloud.tools.intellij.resources.DefaultGoogleApiClientFactory"/>
   </extensions>
 
-  <extensions defaultExtensionNs="com.google.gct.core">
-    <!-- For IJ Community Edition -->
-    <usageTracker implementationClass="com.google.cloud.tools.intellij.analytics.GoogleUsageTracker"
-                  key="Idea"/>
```

Comment:
this is the mechanism we used to whitelist platforms for usage tracking:

`Idea` is the platform key for Community
`idea` is the platform key for Ultimate

Reply:
I think this was originally to differentiate betweeen android studio and intellij. And I think at the time, I should've just done `Android Studio` and `default=GoogleUsageTracker` but for some reason, which I cannot remember now, I set `default=NoOp` and explicitly whitelisted `Idea` and `idea`. Whitelist is probably unnecessary though.

Reply:
do you mean just key by `AndroidStudio` with an implementationClass of `NoOp`. And then default to the GoogleUsageTracker when no matching key is found?

I actually was going to do that before I went down this road. Maybe its better now that I think about it (just in case we need a configuration based blacklist option).